CombineUnloadManager.lua

```diff
@@ -54,7 +54,7 @@ function CombineUnloadManager:addNewCombines()
 		-- this isn't needed as combines will be added when an CombineAIDriver is created for them
 		-- but we want to be able to reload this file on the fly when developing/troubleshooting
 		for _, vehicle in pairs(g_currentMission.vehicles) do
-			if vehicle.cp.driver and vehicle.cp.driver:is_a(CombineAIDriver) and not self.combines[vehicle] then
+			if vehicle.cp.driver and vehicle.cp.driver.is_a and vehicle.cp.driver:is_a(CombineAIDriver) and not self.combines[vehicle] then
```

Comment:
I remember these errors but still have no idea how can is_a not be defined if there is a driver.

Reply:
might have something to do with this one :
 `status,driver,err,errDriverName = xpcall(AIDriver, function(err) printCallstack(); return self,err,"AIDriver" end, vehicle)`

Reply:
in `courseplay:setAIDriver(vehicle, mode)`